request/patch.go

```diff
@@ -48,6 +60,32 @@ type Patch struct {
 	Value interface{} `json:"value"`
 }
 
+// GetPatches gets the patches from the request body and returns it in the form of []Patch
```

Comment:
Can you expand the doc to state: Returns error if 
- request body cannot be read.
- Unmarshalling the requets body is unsuccessful
- No patches are provided in the request.
- Any of the provided patches are invalid.
Just makes it really clear to anyone using it the exact functionality to expect.